### PR TITLE
unordered updates

### PR DESCRIPTION
**Pull request by nwatx**
add basic overview operations for unordered_map among some small changes.

### CODE REVIEW

**Review by HarryWangATX — APPROVED** [review]

File: content/4_Gold/Unordered.mdx [hunk 1]
```diff
@@ -31,14 +31,15 @@ operations. However, as the name implies, elements are not ordered in any
...
+hashed does not require custom hashing.
 
-<IncompleteSection>
-
-Introduce `unordered_set`, `unordered_map`, but no need to repeat code from
-previous module.
+The operations for `unordered_map` include:
 
-</IncompleteSection>
+- `insert`, `erase`, `count`, which are the same as
```
Comment (dongliu0426) [inline content/4_Gold/Unordered.mdx:40]:
```suggestion
- `insert`, `erase`, `count`, which function the same as
```

File: content/4_Gold/Unordered.mdx [hunk 2]
```diff
@@ -31,14 +31,15 @@ operations. However, as the name implies, elements are not ordered in any
...
+  [sorted sets](/bronze/intro-sets)
+- `find`, which gets the iterator to an element.
```
Comment (dongliu0426) [inline content/4_Gold/Unordered.mdx:42]:
```suggestion
- `find`, which returns the iterator to an element.
```

File: content/4_Gold/Unordered.mdx [hunk 3]
```diff
@@ -31,14 +31,15 @@ operations. However, as the name implies, elements are not ordered in any
 meaningful way, so traversals of an unordered set will return elements in some
 arbitrary order.
 
-(more in-depth explanation?)
+Oftentimes, `unordered_map` and `unordered_set` can be used as a drop-in
+replacement for their `map` and `set` counterparts, as long as the type being
+hashed does not require custom hashing.
```
Comment (dongliu0426) [inline content/4_Gold/Unordered.mdx:36]:
```suggestion
Oftentimes, `unordered_map` and `unordered_set` can be used as a drop-in
replacement for their `map` and `set` counterparts, as long as the key does not require custom hashing.
```

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

File: content/4_Gold/Unordered.mdx [hunk 4]
```diff
@@ -31,14 +31,14 @@ operations. However, as the name implies, elements are not ordered in any
...
 
-<IncompleteSection>
-
-Introduce `unordered_set`, `unordered_map`, but no need to repeat code from
-previous module.
+The operations for `unordered_map` include:
 
-</IncompleteSection>
+- `insert`, `erase`, `count`, which all function the same as
+  [sorted sets](/bronze/intro-sets)
```
Comment (dongliu0426) [inline content/4_Gold/Unordered.mdx:40]: add period

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**